runtime/tutor

```diff
@@ -834,6 +834,28 @@ lines.
 
 
 
+=================================================================
+=         7.5 PROGRESSIVE INCREMENTING AND DECREMENTING         =
```

Comment:
What does "progressive" mean here? I don't see it used in the [keymap doc](https://github.com/helix-editor/helix/blob/70a9477ec8221f25e7ee978f0fa0217f98c861c6/book/src/keymap.md#L97-L98) or the [source](https://github.com/search?q=repo%3Ahelix-editor%2Fhelix%20progressive&type=code). If it's redundant with the concept of incrementing, I find that confusing and it would be better to remove it. Luckily everything works fine grammatically if you just remove it from all three spots.

Reply:
https://marketplace.visualstudio.com/items?itemName=narsenico.vscode-progressive-increment#review-details

I took got the progressive part from this VSCode extension. I thought that it was standard terminology.